source/client.js

```diff
@@ -13,7 +15,11 @@ const pageDetails = parse(window.location.search)
 
 render(
   <div>
```

Comment:
I don't like the current default so I think rendering an empty div to start? Not very progressive though ... I'll think about it while I work on the email template across email clients